### PR TITLE
Add tests to ModalError component

### PR DESCRIPTION
**Pull request by caiovictorpcb**
# Description

- Add test to the ModalError component

## Does this pull request close any open issues?

- closes (https://github.com/RibonDAO/interface/issues/461)

## Type of change

Please delete options that are not relevant.

- [X] Tests

image

Only function with no coverage is the default void function created if onClose is not passed on props, hope this is acceptable 😅
image

### CODE REVIEW

File: src/components/moleculars/modals/ModalError/styles.ts [hunk 1]
```diff
@@ -27,11 +27,6 @@ export const Body = styled.h4`
   color: ${({ theme }) => theme.colors.neutral[500]};
 `;
 
-export const RowsModalRow = styled.div`
-  margin-bottom: ${({ theme }) => theme.spacing(32)};
-  display: flex;
-`;
-
```
Comment (caiovictorpcb) [inline src/components/moleculars/modals/ModalError/styles.ts:34]: Remove unused/duplicate styled component.

File: src/setupTests.ts [hunk 2]
```diff
@@ -28,6 +29,10 @@ export function setupMocks() {
     logPageView: mockLogPageViewFunction,
     newLogEvent: mockNewLogEventFunction,
   }));
+  jest.mock("config/zendesk/features", () => ({
+    __esModule: true,
+    ZendeskOpenChat: mockZendeskOpenChatFunction,
+  }));
```
Comment (caiovictorpcb) [inline src/setupTests.ts:35]: Mock zandesk open chat function so i can track it on test :)

File: src/components/moleculars/modals/ModalError/index.test.tsx [hunk 3]
```diff
@@ -64,4 +65,25 @@ describe("ModalError", () => {
...
     });
   });
+
+  describe("when the modal is visible and clicks on button", () => {
+    it("open zendesk chat clicking button", () => {
+      renderComponent(
+        <ModalError visible  />,
+      );
+      const button = screen.getByRole("button")
+      button.click()
```
Comment (NickNish09) [inline src/components/moleculars/modals/ModalError/index.test.tsx:75]: maybe here I would use the `clickOn` helper with the label of the support button. What do you think?
```
clickOn("Access user support");
```

Comment (caiovictorpcb) [reply src/components/moleculars/modals/ModalError/index.test.tsx]: yeah, you're right. ty

**Review by thaisribon — APPROVED** [review]

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**